consensus/istanbul/backend/backend.go

```diff
@@ -371,6 +377,16 @@ func (sb *Backend) verifyValSetDiff(proposal istanbul.Proposal, block *types.Blo
 	return nil
 }
 
+// Sign implements istanbul.Backend.Decrypt
+func (sb *Backend) Decrypt(data []byte) ([]byte, error) {
+	if sb.decryptFn == nil {
```

Comment:
I think that we should require that the decryptFn be non nil.

Reply:
Just read this again, and you are doing exactly that :).

Reply:
closing because no longer using decryptfn